pindogramApp/Controllers/MemesController.cs

```diff
@@ -17,92 +21,117 @@ namespace pindogramApp.Controllers
     public class MemesController : ControllerBase
```

Comment:
Is you are using service in this case 'MemeService' then all action on databse should be done in this service. Remove _context object from MemeController and move the logic to this service